src/side-nav.scss

```diff
@@ -107,6 +111,10 @@ $has-child-content-arrow-margin: 0.125rem;
           }
         }
       }
+
+      &__popover-header {
+        box-shadow: var(--sapContent_HeaderShadow);
+      }
```

Comment:
can you update with an example the Condensed state so we catch problems like this in the visual regression tests? 
Where did you take this value from?

Reply:
@InnaAtanasova 
on closely checking in Ngx, i don't find this class(`fd-nested-list__popover-header`) applied and there is no effect of adding/removing this class on any html element.
<img width="1671" alt="Screenshot 2021-07-12 at 3 11 05 PM" src="https://user-images.githubusercontent.com/57661487/125273834-f5637680-e32a-11eb-9bb1-56e9b29d7e53.png">

In Ngx, value has been used for popover-header-shadow. In styles i used corresponding variable from 
`https://github.com/SAP/theming-base-content/blob/master/content/Base/baseLib/sap_fiori_3/css_variables.css`.

This class (`fd-nested-list__popover-header`) can be removed safely from Ngx and does not need to be moved to Styles.